tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+build method with the following highlighted code. This removes the
+Scaffold and AppBar instances from MyApp. These are now managed
+by RandomWordsState and make it possible to navigate from one screen
+to another in the next step.
+
+Make the changes as shown below:
+
+<!-- skip -->
+{% prettify dart %}
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    [[highlight]]return new MaterialApp([[/highlight]]
+      [[highlight]]title: 'Startup Name Generator',[[/highlight]]
+      [[highlight]]home: new RandomWords(),[[/highlight]]
+    );
+  }
+}
+{% endprettify %}
+</li>
+
+</ol>
+
+Run the app. You should see a list of word pairings. Scroll down
+as far as you want and you will continue to see new word pairings.
+
+### Problems?
+
+If you've gotten off track, you can use the following
+**lib/main.dart** code to get back on track.
+
+{% prettify dart %}
+import 'package:flutter/material.dart';
+import 'package:english_words/english_words.dart';
+
+void main() => runApp(new MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Startup Name Generator',
+      home: new RandomWords(),
+    );
+  }
+}
+
+class RandomWords extends StatefulWidget {
+  @override
+  createState() => new RandomWordsState();
+}
+
+class RandomWordsState extends State<RandomWords> {
+  final _suggestions = <WordPair>[];
+
+  final TextStyle _biggerFont = new TextStyle(fontSize: 18.0);
+
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      appBar: new AppBar(
+        title: new Text('Startup Name Generator'),
+      ),
+      body: _buildSuggestions(),
+    );
+  }
+
+  Widget _buildSuggestions() {
+    return new ListView.builder(
+        padding: const EdgeInsets.all(16.0),
+        itemBuilder: (context, i) {
+          if (i.isOdd) return new Divider();
+
+          final index = i ~/ 2;
+          if (index >= _suggestions.length) {
+            _suggestions.addAll(generateWordPairs().take(10));
+          }
+          return _buildRow(_suggestions[index]);
+        });
+  }
+
+  Widget _buildRow(WordPair pair) {
+    return new ListTile(
+        title: new Text(
+      pair.asPascalCase,
+      style: _biggerFont,
+    ));
+  }
+}
+{% endprettify %}
+
+---
+
+## Step 5: Save favorite word pairings
+
+In this step, you'll add tappable heart icons to each row.
+When the user taps an entry in the list, toggling its
+"favorited" state, that word pairing is added or removed from a
+set of saved favorites.
+
+<!--
+- Add ability to save favorites (in state class)
+- Add navigation to view saved word pairings
+-->
+
+<ol markdown="1">
+<li markdown="1"> Add a `_saved` Set to RandomWordsState. This Set stores
+    the word pairings that the user favorited. A Set is used instead of a List
+    because a Set typically doesn't allow duplicates, if it's implemented not to
```

Comment:
a set definitely doesn't allow duplicates, suggest we remove the word 'typically'

Reply:
Done